### ISSUE TITLE
[Snap] Fix bash completion

### ISSUE THREAD

**Issue by BoboTiG** [httpie/httpie#1107]
Bash completion currently does not work in the Snap package. See how it could be fixed.
Interesting links:
- https://snapcraft.io/docs/tab-completion-for-snaps
- https://forum.snapcraft.io/t/debugging-tab-completion/4198

The first thing is to adapt that line: https://github.com/httpie/httpie/blob/b7300c1096796dce127152016dbc138145bf1001/extras/httpie-completion.bash#L13
It must be changed to:
```bash
complete -o default -F _http_complete http https httpie.http httpie.https
```
And then, debug why it does not work :)

**Comment by BoboTiG** [comment]
That would be cool :)

**Comment by this-is-r-gaurav** [comment]
Hi @BoboTiG Just to keep track of it,  I tried to debug that, the command `complete -p httpie.http` is failing with `no completion specification`.
I was trying to figure it out why specification is not loading, but not able to figure it out. I might take another weekend, to debug that. 
____
with  command
```bash
snap run --command=complete httpie.http 9 9 11 1 " " 'httpie.http -' httpie.http -
```
It is listing all options.

**Comment by isidentical** [comment]
This seems to be resolved by #1189. Let us know if there is anything else to be done by creating a new issue.
